spec/spec_helper.rb

```diff
@@ -69,6 +69,10 @@
   # rspec-rails.
   config.infer_base_class_for_anonymous_controllers = false
 
+  # TODO - remove this line after Spree upgrade. Current Spree specs do not have `type: :controller` definitions.
+  # This is a workaround to pass tests in Travis i.e. https://travis-ci.org/openfoodfoundation/openfoodnetwork/jobs/235601640
+  config.infer_spec_type_from_file_location!
+
```

Comment:
I tried this in https://github.com/jeronimo/openfoodnetwork/compare/rspec-upgrade...mkllnk:rspec-upgrade. But it actually made a spec fail: https://travis-ci.org/mkllnk/openfoodnetwork/jobs/248648740

It's somehow loading a Spree config instead of the OFN config (change of host in url). Any ideas?

@oeoeaio and @enricostano What don't you like about inferring the spec type from the location? Is it considered bad style?